setup-docker.py

```
@@ -0,0 +1,39 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
```

Ruff (0.15.0)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)